cni/log/logger.go

```diff
@@ -17,6 +17,8 @@ var (
 const (
 	maxLogFileSizeInMb = 5
 	maxLogFileCount    = 8
+	ETWCNIEventName    = "Azure-CNI"
```

Comment:
nit:
```suggestion
	etwCNIEventName    = "Azure-CNI"
```
capital letters mean `public` in Go, but this is not used outside of this package